Tests/BasicTests/ResultTests.swift

```diff
@@ -140,52 +87,28 @@ class ResultTests: XCTestCase {
         }
 
         // We should be able to map when we have value in result and our closure doesn't throw.
-        let success = Result<String, AnyError>("Hello").mapAny { value -> String in
+        let success = Result<String, AnyError>.success("Hello").mapAny { value -> String in
             let second = try throwing(false)
             return value + second
         }
-        XCTAssertEqual(try success.dematerialize(), "Hello World")
+        XCTAssertEqual(try success.get(), "Hello World")
 
         // We don't have a value, closure shouldn't matter.
         let failure1 = Result<String, AnyError>(DummyError.somethingWentWrong).mapAny { value -> String in
             let second = try throwing(false)
             return value + second
         }
         XCTAssertThrowsAny(DummyError.somethingWentWrong) {
-            _ = try failure1.dematerialize()
+            _ = try failure1.get()
         }
 
         // We have a value, but our closure throws.
-        let failure2 = Result<String, AnyError>("Hello").mapAny { value -> String in
+        let failure2 = Result<String, AnyError>.success("Hello").mapAny { value -> String in
             let second = try throwing(true)
             return value + second
         }
         XCTAssertThrowsAny(DummyError.somethingWentWrong) {
-            _ = try failure2.dematerialize()
-        }
-    }
-
-    func testFlatMap() throws {
```

Comment:
You'll need to update the LinuxMain file since we're changing the tests. You can do that by running `.build/x86_64-apple-macosx/debug/swift-test --generate-linuxmain`